### ISSUE TITLE
Notification (email) when user joins team / RSVPs for an event

### ISSUE THREAD

**Issue by craigmaloney** [GetTogetherComm/GetTogether#188]
As a team moderator I would love a notification when a user joins my team and RSVPs for an event. I know that not everyone would like this so having this be selectable (possibly at the profile / team level) would be great.

**Comment by mhall119** [comment]
You should already see these messages if you're an admin. Do you think they need to be sent to all moderators as well?

**Comment by craigmaloney** [comment]
Send to Moderators: Maybe as an option for the group perhaps?

Hm, wondering if this is specific to my mail server / your mail service rejecting the messages.

**Comment by craigmaloney** [comment]
Ah, it looks like I got the email on the next day, not when the member joins. 

Wondering if that might be something more useful on an hourly basis rather than a daily basis?

**Comment by mhall119** [comment]
Hmmm, maybe. The idea was to limit the number of emails sent out. So if you had 10 people join in a day, you'd get 1 email with all of their names, rather than 10 separate emails

**Comment by craigmaloney** [comment]
Right, however I got notifications for RSVPs for a meeting that happened yesterday.

Perhaps a digest option for those who don't want to know straight-away?

**Comment by mhall119** [comment]
I'm avoiding making things options as much as possible, because the vast majority of users don't seem to change from whatever is the default. I'd prefer to find something that "just works" as desired in nearly all cases.

A simple fix would be to bump up the frequency of attendee notification emails to twice daily, of even every 6 hours. 

A more complicated, but perhaps better, solution would be to increase frequency as the event time gets closer, so you'd get daily emails until 24 or 48 hours before the event, and then every 6 hours after that. Maybe even hourly emails in the last 6 hours? Deciding on the rate change would likely be harder than the actual implementation of this.
